core/jobs.py

```diff
@@ -364,6 +364,20 @@ def _real_enqueue(cls, job_id, queue_name, additional_job_params):
         raise NotImplementedError(
             'Subclasses of BaseJobManager should implement _real_enqueue().')
 
+    @classmethod
+    def real_enqueue(cls, job_id, queue_name, additional_job_params):
+        """Public method to test the private method
```

Comment:
`Public wrapper for the _real_enqueue method`

Reply:
I have fixed the doc